handlers/server.go

```diff
@@ -0,0 +1,102 @@
+package handlers
+
+import (
+	"fmt"
+	"github.com/gorilla/mux"
+	"github.com/shayan-golafshani/golang-http-api-w-AWS/store"
+	"log"
+	"net/http"
+	"time"
+)
+
+type Server struct {
+	Router http.Handler
+	Db     store.SubsetDynamoDb
+}
+
+func (s *Server) SetupRouter() {
+	s.Router = GetRouter(s)
+}
+
+const (
+	TableName = "Employees"
+)
+
+// GetRouter builds the mux router used by a server
+func GetRouter(server *Server) *mux.Router {
+	fmt.Println("getting router on Lambda")
+	//ADD EMPLOYEES TO THE STORE... will need to store these bad bois in DynamoDB for persistence..
+	//store.AddEmployees()
+
+	//New mux.Router
+	r := mux.NewRouter()
+	//USE logging middleware, w method durations, time-stamps, & define request type
+	//r.Use(CustomMiddleware)
```

Comment:
how would you leverage middleware for stuff like user authentication and authorization?